docs/camino-messenger/bot/configuration.mdx

```diff
@@ -0,0 +1,157 @@
+---
+sidebar_position: 10
+title: Configuration & Running
+description: How to Configure & Run Camino Messenger Bot?
+---
+
+import AsciinemaEmbed from "/src/components/AsciinemaScriptInjector";
+
+# Camino Messenger Bot Configuration
+
+:::info DRAFT DOCUMENTATION NOTICE
+
+Please be aware that this document is currently a draft and is undergoing
+active development. Content, guidelines, and instructions may be subject to
+change.
+
+:::
+
+:::caution 🚧 ALPHA CODE NOTICE 🚧
+
+This application is in the alpha phase of development. It is important to note
+that during this stage, breaking changes may occur without advance notice.
+Users should proceed with caution.
+
+:::
+
+## Copy the Example Configuration File
+
+After you've installed Camino Messenger Bot (CMB), copy the example
+configuration file `camino-messenger-bot.yaml.example`, removing the
+`.example` at the end.
+
+```sh
+cp camino-messenger-bot.yaml.example camino-messenger-bot.yaml
+```
+
+**Example Bot Config File:**
+
+```yaml
+# Example config
+developer_mode: true
+matrix_key: PrivateKey-YOUR_PRIVATE_KEY
+matrix_host: matrix.camino.network
+matrix_store: mautrix.db
+rpc_server_port: 9090
+rpc_unencrypted: false
+rpc_server_cert_file: server-cert.pem
+rpc_server_key_file: server-key.pem
+partner_plugin_host: localhost
+partner_plugin_port: 50051
+partner_plugin_unencrypted: false
+partner_plugin_ca_file: ca-cert.pem
+response_timeout: 3000
+supported_request_types:
+  - ActivitySearchRequest
+  - AccommodationSearchRequest
+  - GetNetworkFeeRequest
+  - GetPartnerConfigurationRequest
+  - PingRequest
+  - TransportSearchRequest
+```
+
```

Comment:
Two notes here:
- it would make sense to add a hint here that the `partner_plugin_port` value should match of the port described here: https://playground.docs.camino.network/camino-messenger/bot/partner-plugin#running-the-partner-plugin
- also adding an explanation regarding `supported_request_types` makes sense I believe. These types are relevant only when running a provider bot and the provider can choose which requests to support. That is, the provider is not required to support all the types.

Reply:
Added it to a note block below the section.